src/Sentry/Internal/Polyfills.cs

```diff
@@ -2,15 +2,16 @@
 
 // Polyfills to bridge the missing APIs in older versions of the framework/standard.
 // In some cases, these just proxy calls to existing methods but also provide a signature that matches .netstd2.1
-#if NET461 || NETSTANDARD2_0
 
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
```

Comment:
Without having the using directive, when using "refactoring" it deletes them as it doesn't know it's used by the code currently not compiling due to #ifdef

ideally only what's needed is available

Reply:
I know, but I haven't been able to come up with a better way :/
There are two different #if regions and they have different usings.
Previously I was able to scope it to a class, but these polyfills need to be in global namespace because they fill intrinsic type members, which would normally not require adding a using on the calling side.

Reply:
Can probably add a ReSharper ignore comment about unused directives? Maybe that will stop the refactor from touching this file.

Reply:
Yeah, that appears to work.

Reply:
you can just add `#if` and the right `using` under it. But no strong feelings about this, if it doesn't get dropped because of a refactor, it's OK.